integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy

```diff
@@ -232,13 +232,13 @@ class UsersIT extends ITSupport implements CrudTestSupport {
         Role role = user.assignRole(assignRoleRequest)
 
         // 3. List roles for the user and verify added role
-        assertPresent(user.listRoles(), role)
+        assertPresent(user.listAssignedRoles(), role)
```

Comment:
`listAssignedRoles()` seems odd in this context cause there isn't other types of roles that can be associated with a user